src/lib/cli/rules/common.ts

```diff
@@ -41,6 +97,166 @@ export type NewRule = {
     events: string;
     /** If the new rule is core */
     isRecommended: boolean;
-    /**  Usage categories that the new rule applies to */
+    /** Usage categories that the new rule applies to */
     useCase?: UseCase;
+    /** Path prefix for dependencies */
+    prefix: string;
+    /** If the rule works with local files */
+    worksWithLocalFiles: Boolean;
+    /** If a rule is external */
+    external: Boolean;
+}
+
+export enum RuleType {
+    core = 'core',
+    external = 'external',
+    externalRule = 'externalRule'
```

Comment:
@sarvaje Can you specify what is the difference between `external` and `externalRule`? Thanks!

Reply:
ok!

Reply:
@alrra I'm going to change the name of the `enum` to `QuestionsType` to be more accurate.